pkg/atmos/component-helper/20_enabled_flag_tst.go

```diff
@@ -25,8 +26,8 @@ func (s *TestSuite) VerifyEnabledFlag(componentName, stackName string, additiona
 
 	atmosOptions := getAtmosOptions(s.T(), s.Config, componentName, stackName, &mergedVars)
 
-	code, err := atmos.PlanExitCodeE(s.T(), atmosOptions)
+	ouputs, err := atmos.PlanE(s.T(), atmosOptions)
 	require.NoError(s.T(), err)
-
-	require.Equal(s.T(), 0, code)
+	noChanges := strings.Contains(ouputs, "No changes. Your infrastructure matches the configuration.") || strings.Contains(ouputs, "without changing any real infrastructure.")
+	require.True(s.T(), noChanges)
 }
```

Comment:
I'm not sure why we'd want to make these changes to rely on "magic strings" rather than the exit code, that they publish in the [docs](https://developer.hashicorp.com/terraform/cli/commands/plan#other-options).

Reply:
Since you added the `DriftTest` you can just make this call `DriftTest` with `enabled=false` as the additional variable.